MAUI/NumericEntry/Restriction.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: Value change restriction in .NET MAUI NumericEntry | Syncfusion
+description: Learn here all about how to restrict the value change in Syncfusion .NET MAUI NumericEntry (SfNumericEntry) control and more.
+platform: MAUI
+control:  SfNumericEntry
+documentation: ug
+---
+
+# Value change restriction in .NET MAUI NumericEntry
+
+This section describes how to restrict the change in value of the `NumericEntry` control using `AllowNull`, `Minimum`, and `Maximum` properties.
```

Comment:
This section describes how to restrict the change in the value of the `NumericEntry` control using `AllowNull`, `Minimum`, and `Maximum` properties.

Reply:
The requested content modification has been successfully addressed